src/model.ts

```diff
@@ -518,6 +523,42 @@ export function build<TSchema extends BaseSchema, TDefaults extends Partial<TSch
     }
   );
 
+  /**
+   * @description
+   * Performs an optimized `find` to test for the existence of any document matching the filter criteria.
+   *
+   *
+   * @param filter {Filter<TSchema>}
+   * @param [options] {Omit<FindOptions<TSchema>, "projection" | "limit" | "sort" | "skip">}
+   *
+   * @returns {Promise<boolean>}
+   *
+   * @example
+   * const isAlreadyActive = await User.exists({ firstName: 'John', lastName: 'Wick', active: true });
```

Comment:
One small change request: break this object on multiple lines, so that it reads nicely on the docs page.